Background.md

```diff
@@ -0,0 +1,47 @@
+
+Data spaces lijken meer gericht te zijn op organisaties met als doel het stimuleren en bereiken van data delen/samenwerken en tegelijkertijd controle te houden over je data.
+
+Binnen dit project zal de focus meer liggen op data pods.
+
+## WebID
+
+Een WebID duidt op meerdere 'dingen'. Het is:
+
+1. een unieke IRI (International Resource Identifier) en URL
+1. een document dat een 'resource' beschrijft; dat kan een beschrijving van de mens zijn, zijn profiel dus, maar ook een organisatie, een service (bot)
+1. een service voor authenticatie en deels autorisatie (OpenID Connect / OAuth2)
+
+## Verifiable Credentials (VCs)
+
+Verifiable Credentials (VCs) zijn een open standaard voor digitale attributen/referenties. Op de pagina van [W3C](https://www.w3.org/TR/vc-data-model/) kan je veel informatie vinden over VCs. Hun definitie van een VC is als volgt: "A verifiable credential is a tamper-evident credential that has authorship that can be cryptographically verified." Waarbij een credential een "set of one or more claims made by an issuer" is. Vanuit een VC kan een Verifiable Presentation worden gemaakt. 
+
+Schematische weergave
+![image](https://user-images.githubusercontent.com/62643510/200321656-9847af00-7ecb-42ad-bb47-5f53c7957a03.png)
+
+### VCs vs IRMA
+
+[IRMA](https://irma.app/) is een product (app) waarbinnen je attributen kunt laden en daar zelf controle over hebt. IRMA werkt met "disclosure proof" die lijken op VCs (zie hoofdstuk [Cryptographic Entities](https://irma.app/docs/overview/#cryptographic-entities). De 'disclosure proofs' maken IRMA inderdaad verifieerbaar, maar is niet helemaal hetzelfde als een VC. Waar het hier bij beiden over gaat is de data integriteit, het kunnen verifiëren dat je niet zelf aan de attributen loopt te sleutelen. Binnen VC signeert de issuer de credential die die uitgeeft. Iedereen kan die vervolgens verifiëren op basis van de meegeleverde proof, meestal een verwijzing naar de public key van de issuer. Binnen IRMA is het de IRMA server zelf die een signature toevoegd, de disclosure proof. Dat maakt het ook dat non-IRMA applicaties niet de IRMA attributen kan verifieren, maar alleen IRMA servers dat zelf kunnen, in tegenstelling tot VC. 
+
+![image](https://user-images.githubusercontent.com/62643510/200325252-2b110df9-95e4-439f-845b-dc7808a23d4a.png)
```

Comment:
Hier ook het plaatje exporteren, opslaan en toevoegen als bestand en dáár dan naar verwijzen in deze Markdown ...